recipes/pyode/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# defining package and version
+{% set name = "pyode" %}
+{% set version = "0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/odedevs/ode/downloads/ode-{{ version }}.tar.gz
+  sha256: 4ba3b76f9c1314160de483b3db92b0569242a07452cbb25b368e75deb3cabf27
+
+build:
+  number: '0'
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - expat
+    - python {{ PY_VER }}
```

Comment:
What are your intention with this jinja variable?

Reply:
Python should be jus in ``host`` as well

Reply:
OK, removed it, has no effect.